contrib/pg_tde/t/013_crash_recovery.pl

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+use File::Basename;
+use Test::More;
+use lib 't';
+use pgtde;
+
+PGTDE::setup_files_dir(basename($0));
+
+my $node = PostgreSQL::Test::Cluster->new('main');
+$node->init;
+$node->append_conf('postgresql.conf', "shared_preload_libraries = 'pg_tde'");
```

Comment:
To do a recovery tests shouldn't you set the checkpoint timeout to like 60 minutes to make it clear? Or is that overkill?